src/installation/guides/chroot.md

```diff
@@ -0,0 +1,235 @@
+distribution, edit `/mnt/etc/default/libc-locales`, uncommenting desired
+[locales](../../config/locales.md).
+
+[nvi(1)](https://man.voidlinux.org/nvi.1) is available in the chroot, but you
+may wish to install your preferred text editor at this time.
+
+For glibc builds, generate locale files with
+
+```
+(chroot) # xbps-reconfigure -f glibc-locales
+```
+
+[Configure at least one super user account](../../config/users-and-groups.md).
+Other user accounts can be configured later, but there should either be a root
+password, or a new user account with [sudo(8)](https://man.voidlinux.org/sudo.8)
+privileges.
+
+To set a root password, run
+
+```
+(chroot) # passwd
+```
+
+Configure [fstab(5)](https://man.voidlinux.org/fstab.5). This file can be
+automatically generated from currently mounted filesystems by copying the file
+`/proc/mounts`.
+
+```
+(chroot) # cp /proc/mounts /etc/fstab
+```
+
+Remove lines in `/etc/fstab` that refer to `proc`, `sys`, `devtmpfs` and `pts`.
+
+Replace references to `/dev/sdXX` with their respective UUID, which can be found
+by running `lsblk -f` (see [lsblk(8)](https://man.voidlinux.org/lsblk.8)).
+
+Change the last zero of the entry for `/` to `1`, and the last zero of every
+other line to `2`. These values configure the behaviour of
+[fsck(8)](https://man.voidlinux.org/fsck.8).
+
+For example, the partition scheme used throughout previous examples yields the
+following fstab:
+
+```
+/dev/sda1       /boot/EFI   vfat    rw,relatime,[...]       0 0
+/dev/sda2       /           ext4    rw,relatime             0 0
+```
+
+The fstab is modified thus with information from `lsblk -f`:
+
+```
+UUID=6914[...]  /boot/EFI   vfat    rw,relatime,[...]       0 2
+UUID=dc1b[...]  /           ext4    rw,relatime             0 1
+```
+
+> Note: The output of `/proc/mounts` will have a single space between each
+> field. The columns are aligned here for readability.
+
+Add an entry to mount `/tmp` in ram:
+
+```
+tmpfs           /tmp        tmpfs   defaults,nosuid,nodev   0 0
+```
+
+If using swap space, add an entry for any swap partitions:
+
+```
+UUID=1cb4[...]  swap        swap    rw,noatime,discard      0 0
+```
+
+## Installing GRUB
+
+**On a BIOS computer**, install the package `grub`, then run `grub-install
+/dev/sdX` where `/dev/sdX` is the drive (not partition) that you wish to install
+GRUB to.
+
+```
+(chroot) # xbps-install grub
+(chroot) # grub-install /dev/sda
+```
+
+**On a UEFI computer**, install either `grub-x86_64-efi` or `grub-i386-efi`
+depending on your architecture then run `grub-install`, optionally specifying a
+bootloader label (this label may be used by your computer's firmware when
+manually selecting a boot device)
+
+```
+(chroot) # xbps-install grub-x86_64-efi
+(chroot) # grub-install --bootloader-id="Void"
+```
+
+If installing onto a removable disk (such as USB) add the option `--removable`
+to the `grub-install` command.
+
```

Comment:
GRUB is a common bootloader, but Void also offers `gummiboot` and `rEFInd` that can be easier to administer. Eventually, it might be nice to add sections documenting these alternatives. (Of the three, I like GRUB the least.)

Reply:
I'm waiting for these PRs to land to document some dracut UEFI stuff, as well as add some gummiboot and rEFInd info. rEFInd and gummiboot aren't very Void specific, I think, so I don't know how much about it we should mention. dracut definitely feels like it's lacking a "guide", so I'm thinking of putting one here.

Reply:
Don't know about rEFInd, but void has some special tooling around gummiboot to help automate config. This is documented in the install message though.

Reply:
GRUB is also one of my least favourite bootloaders too, but I chose to document it here because it's what you get when you use the live installer. We should probably be discussing next steps over here -> https://github.com/void-linux/void-docs/issues/85